apps/docs/customization/proxy/api-reference.md

```
@@ -0,0 +1,145 @@
+---
+title: Proxy — API reference
+description: Exports from @authhero/proxy, the kysely/drizzle/aws proxy data adapters, and the authhero control-plane KV publisher.
+---
+
+# API reference
+
+
+### From `@authhero/proxy`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)